content/docs/intro/concepts/runtime-functions.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Runtime Functions"
+meta_desc: A quick look at runtime helper functions for Pulumi.
+menu:
+  intro:
+    parent: concepts
+    weight: 10
+---
+
+The Pulumi SDK library includes a number of helper functions for the following actions:
+
+- Getting information about the current deployment
+- Using logging information for diagnostics, warnings, or errors pertaining to the deployment
+- Serializing Lambdas to turn JavaScript callbacks into data, which can be used as application code
+
+## Getting the Current Project or Stack
+
+The [`get_project`]({{< relref "/docs/reference/pkg/python/pulumi#pulumi.get_project" >}}) and [`get_stack`]({{< relref "/docs/reference/pkg/python/pulumi#pulumi.get_stack" >}}) functions give you the currently deployed project and stack, respectively. This can be useful for naming or tagging resources.
+
+```python
+project = pulumi.get_project()
+stack = pulumi.get_stack()
+```
```

Comment:
Looks like we lost all the language choosers here?

Reply:
Fixed in 0f0a3f6.